### ISSUE TITLE
Warning spawning whenever calling every command

### ISSUE THREAD

**Issue by SoftEngie** [fspoettel/advent-of-code-rust#64]
When running the command cargo scaffold 1, 

I get lot of warnings that are making the output unreadable
```
warning: field `0` is never read
  --> src/template/readme_benchmarks.rs:12:12
   |
12 |     Parser(String),
   |     ------ ^^^^^^
   |     |
   |     field in this variant
   |
   = note: `Error` has a derived impl for the trait `Debug`, but this is intentionally ignored during dead code analysis
   = note: `#[warn(dead_code)]` on by default
help: consider changing the field to be of unit type to suppress this warning while preserving the field numbering, or remove the field
   |
12 |     Parser(()),
   |            ~~

warning: field `0` is never read
  --> src/template/readme_benchmarks.rs:13:8
   |
13 |     IO(io::Error),
   |     -- ^^^^^^^^^
   |     |
   |     field in this variant
   |
   = note: `Error` has a derived impl for the trait `Debug`, but this is intentionally ignored during dead code analysis
help: consider changing the field to be of unit type to suppress this warning while preserving the field numbering, or remove the field
   |
13 |     IO(()),
   |        ~~

warning: field `0` is never read
  --> src/template/run_multi.rs:52:8
   |
52 |     IO(io::Error),
   |     -- ^^^^^^^^^
   |     |
   |     field in this variant
   |
   = note: `Error` has a derived impl for the trait `Debug`, but this is intentionally ignored during dead code analysis
help: consider changing the field to be of unit type to suppress this warning while preserving the field numbering, or remove the field
   |
52 |     IO(()),
   |        ~~
```

**Comment by fspoettel** [comment]
@SoftEngie which version of Rust are you using? I haven't updated the template in a while, this looks like a new clippy lint being applied.

**Comment by SoftEngie** [comment]
My version of clippy is : 
$ cargo clippy --version
clippy 0.1.81 (eeb90cd 2024-09-04)

**Comment by fspoettel** [comment]
Thank you for the report, resolved the issues via [e2062e3](https://github.com/fspoettel/advent-of-code-rust/commit/e2062e33fdb48094937c28d64517ff2c52aede05)
